associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl

```diff
@@ -43,196 +50,292 @@
     <category citation-format="author-date"/>
     <category field="generic-base"/>
     <summary>The Brazilian Standard Style in accordance with ABNT-NBR 10520.2002 and ABNT-NBR 6023.2018</summary>
-    <updated>2020-03-06T09:31:45+00:00</updated>
+    <updated>2020-10-06T01:42:52+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="pt-BR">
     <terms>
-      <term name="month-01" form="short">jan.</term>
-      <term name="month-02" form="short">fev.</term>
-      <term name="month-03" form="short">mar.</term>
-      <term name="month-04" form="short">abr.</term>
-      <term name="month-05" form="short">maio</term>
-      <term name="month-06" form="short">jun.</term>
-      <term name="month-07" form="short">jul.</term>
-      <term name="month-08" form="short">ago.</term>
-      <term name="month-09" form="short">set.</term>
-      <term name="month-10" form="short">out.</term>
-      <term name="month-11" form="short">nov.</term>
-      <term name="month-12" form="short">dez.</term>
-      <!--ABNT 6023:2018 - 8.1.1.4 sempre no singular-->
-      <term name="editor" form="short">org.</term>
-      <!--usando editor de coleção para quando for editor do livro-->
-      <term name="collection-editor" form="short">ed.</term>
+      <term name="editor" form="short">org</term>
+      <!-- ABNT 6023:2018 - 8.1.1.4 sempre no singular -->
+      <term name="collection-editor" form="short">ed</term>
+      <!-- usando editor de coleção para quando for editor do livro -->
+      <term name="accessed">acesso</term>
```

Comment:
a lot of these are already in the locale -- no need to repeat, remove them here, please:
https://github.com/citation-style-language/locales/blob/master/locales-pt-BR.xml

Reply:
The only one that I saw as the same is "avaible at". Removed.